frontend/src/components/TagInput/TagInput.tsx

```diff
@@ -39,9 +39,10 @@ export interface TagInputProps
   keyDownKeys?: string[]
   /**
    * Optional function to call to validate created tags.
-   * Should return `true` if tag is invalid
+   * Should return `true` if tag is valid.
+   * If no function is provided, all tags are considered valid.
    */
-  tagInvalidation?: (tag: string) => boolean
+  tagValidation?: (tag: string) => boolean
```

Comment:
why not rename this `isTagValid`? we normally use `is` or `has` prefix for functions which return boolean right

Reply:
Only for Boolean constants tbh, mixing them for both functions and constants might be confusing (even though I know we don't always follow that)